docs/csharp/tutorials/inheritance.md

```diff
@@ -183,7 +183,7 @@ In designing our `Publication` class, we need to make several design decisions:
 
   By default, a base class can be instantiated by calling its class constructor. Note that we do not have to explicitly define a class constructor. If one is not present in the base class' source code, the C# compiler automatically provides a default (parameterless) constructor.
```

Comment:
nit for line 82: I had to think about what "Otherwise" referred to and read the paragraph several times (it complements the "If" in the second sentence."  I think that the Otherwise sentence can be moved up to follow the If sentence, and the paragraph still reads well.